client/my-sites/site-settings/media-settings/index.jsx

```diff
@@ -0,0 +1,138 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import SectionHeader from 'components/section-header';
+import Card from 'components/card';
+import Button from 'components/button';
+import JetpackModuleToggle from '../jetpack-module-toggle';
+import FormFieldset from 'components/forms/form-fieldset';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+import CarouselSettings from './carousel-settings';
+import { protectForm } from 'lib/protect-form';
+
+import {
+	getJetpackSetting,
+	isRequestingJetpackSettings,
+	isUpdatingJetpackSettings
+} from 'state/jetpack/settings/selectors';
+import { updateSettings } from 'state/jetpack/settings/actions';
+
+class MediaSettings extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			carousel_display_exif: props.carousel_display_exif,
+			carousel_background_color: props.carousel_background_color
+		};
+	}
+
+	componentWillReceiveProps( newProps ) {
+		this.setState( {
+			carousel_display_exif: newProps.carousel_display_exif,
+			carousel_background_color: newProps.carousel_background_color
+		} );
+	}
+
+	onInputChange = ( updated ) => {
+		this.setState( updated );
+		this.props.markChanged();
+	}
+
+	submit = () => {
+		return this.props.updateSettings( this.props.site.ID, {
+			carousel_background_color: this.state.carousel_background_color,
+			carousel_display_exif: this.state.carousel_display_exif
+		} ).then( () => {
+			this.props.markSaved();
+		} );
+	}
+
+	render() {
+		const props = this.props;
+		return (
+			<form id="media-settings" onSubmit={ this.submit }>
+				<SectionHeader label={ props.translate( 'Media' ) }>
+					<Button
+						compact
+						primary
+						onClick={ this.submit }
+						disabled={ props.isRequestingJetpackSettings || props.submittingForm }>
+						{ props.submittingForm ? props.translate( 'Saving…' ) : props.translate( 'Save Settings' ) }
+					</Button>
+				</SectionHeader>
+				<Card className="media-settings__card site-settings">
+					<FormFieldset>
+						<div className="media-settings__info-link-container">
+							<InfoPopover position={ 'left' }>
+								<ExternalLink target="_blank" icon={ true } href={ 'https://jetpack.com/support/photon' } >
+									{ props.translate( 'Learn more about Photon' ) }
+								</ExternalLink>
+							</InfoPopover>
+						</div>
+						<JetpackModuleToggle
+							siteId={ props.site.ID }
+							moduleSlug="photon"
+							label={ props.translate( 'Speed up your images and photos with Photon.' ) }
```

Comment:
Alternate string suggestion: 
* `translate( '<strong>Speed up</strong> your images and photos.' )`&mdash; translations: **39**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/jetpack-site-settings-ui)